pipeline/processor.go

```diff
@@ -10,17 +11,18 @@ type ActionResult int
 
 const (
 	// ActionPass pass event to the next action in a pipeline
-	ActionPass ActionResult = 0
+	ActionPass ActionResult = iota
 	// ActionCollapse skip further processing of event and request next event from the same stream and source as current
 	// plugin may receive event with EventKindTimeout if it takes to long to read next event from same stream
-	ActionCollapse ActionResult = 2
+	ActionCollapse
 	// ActionDiscard skip further processing of event and request next event from any stream and source
-	ActionDiscard ActionResult = 1
+	ActionDiscard
 	// ActionHold hold event in a plugin and request next event from the same stream and source as current.
 	// same as ActionCollapse but held event should be manually committed or returned into pipeline.
 	// check out Commit()/Propagate() functions in InputPluginController.
 	// plugin may receive event with EventKindTimeout if it takes to long to read next event from same stream.
-	ActionHold ActionResult = 3
+	ActionHold
+	ActionSpawned = 4
```

Comment:
Action should be a verb. `Spawned` isn't a verb.

Reply:
Write a comment please about new action

Reply:
Done. Now its `ActionBreak`